apiserver/service/router.go

```diff
@@ -49,12 +52,13 @@ func NewServerAndRun(conf config.ServerConfig) {
 	if conf.EnableOIDC {
 		oidc.InitOIDCArgs(conf.IssuerURL, conf.MasterURL, conf.ClientSecret, conf.ClientID)
 	}
-
 	bffGroup := r.Group("/bff")
 	chatGroup := r.Group("/chat")
 	RegisterMinIOAPI(bffGroup, conf)
 	RegisterGraphQL(r, bffGroup, conf)
-	RegisterChat(chatGroup, conf)
+	registerChat(chatGroup, conf)
+	docs.SwaggerInfo.BasePath = "/"
+	r.GET("swagger/*any", ginSwagger.WrapHandler(swaggerfiles.Handler))
```

Comment:
Should we add a argument to controll whether or not to  expose this swagger api?